tests/test_pointset.py

```diff
@@ -62,3 +62,15 @@ def test_filters_return_pointset(sphere):
     pointset = sphere.cast_to_pointset()
     clipped = pointset.clip()
     assert isinstance(clipped, pyvista.PointSet)
+
+
+def test_volume_area_pointgrid():
+    # 3D grid size 4x4x4
+    grid = pyvista.UniformGrid(dims=(5, 5, 5)).cast_to_unstructured_grid()
```

Comment:
Sorry, one more question: do I understand correctly that these grids are converted to unstuctured grids because `UniformGrid` doesn't inherit from `PointGrid`? But I see the `compute_cell_sizes()` method on `UniformGrid`s. Couldn't we give them the same properties, then? Presumably somewhere higher up the inheritance tree.

Reply:
Despite grid being in the name of both, the common ancestor is `DataSet`.  But you are right this PR aligns a lot of these methods and we can probably move all these up to `DataSet`.